reana-ui/src/components/WorkflowBadges.js

```diff
@@ -0,0 +1,66 @@
+import styles from "./WorkflowBadges.module.scss";
+import { Label } from "semantic-ui-react";
+import { JupyterNotebookIcon } from "~/components";
+import { INTERACTIVE_SESSION_URL } from "~/client";
+import { LauncherLabel } from "~/components";
+import { getReanaToken } from "~/selectors";
+import { useSelector } from "react-redux";
+import { statusMapping } from "~/util";
+
+export default function WorkflowBadges({ workflow }) {
+  const reanaToken = useSelector(getReanaToken);
+  const {
+    id,
+    size,
+    launcherURL,
+    session_uri: sessionUri,
+    session_status: sessionStatus,
+  } = workflow;
+  const hasDiskUsage = size.raw > 0;
+  const isSessionOpen = sessionStatus === "created";
+
+  return (
+    <>
+      <div className={styles.badgesContainer}>
+        {workflow.duration && (
+          <Label
+            size="tiny"
+            content={workflow.duration}
+            icon="clock"
+            as="a"
+            href={"/details/" + id}
+            target="_blank"
```

Comment:
A functional observation. Clicking on a badge on the workflow list page (and on workflow detailed page) leads to opening new tabs. This can quickly fill up, e.g. try to click several times on the duration badge on the worklow detailed page, it'll generate a new tab for every repeated click.

It would be better to always open in the same tab, i.e. remove `_blank` targets here and elsewhere in the code. If a user would like to open a new tab, they could always middle-click.

Reply:
The links on the disk/duration badges were removed. If we want to keep them and to make them point to specific tabs of the workflow detail page, maybe we should open an issue and do so in another PR?